upkie/cpp/actuation/BulletInterface.h

```diff
@@ -12,9 +12,10 @@
 #include <string>
 #include <vector>
 
-#include "RobotSimulator/b3RobotSimulatorClientAPI.h"
+#include "SharedMemory/PhysicsClientC_API.h"
```

Comment:
```suggestion
```
IIUC this header is not used in `BulletInterface.h`. (`RobotSimulator.h` uses it, and it correctly includes it already.)